File: Klasa 2/Python/hello.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-

rok = 2017
data_powstania = 1991

def parzyste(zakres):
    ile = list(range(0, zakres+1, 2))
    print("Liczby parzyste: ")
    print(ile)
    return len(ile)
    
    
def main(args):
    imie = input("Podaj imie: ")
    print ("Witaj", imie)
    wiek = int(input("Ile masz lat? "))
    print ("Urodziłeś się w" , rok-wiek, "roku")
    
    if wiek > (rok-data_powstania):
        print("Jestem młodszy!")
    elif wiek < (rok-data_powstania):
        print("Jestem starszy!")
    else:
        print("Mamy tyle samo lat!")
        
    zakres = int(input("Podaj zakres: "))
    print("Ilosc liczb parzystych: ", parzyste(zakres))
    
    return 0

if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))

File: Klasa 2/Cpp/Hello.cpp
#include <iostream>

using namespace std;

int main(int argc, char **argv)
{
    string imie;
    int wiek, rok = 2017;
    const int data_powstania = 1983;
    int wiek_cpp = rok - data_powstania;
    
    cout << "Podaj imie: ";
    cin >> imie;
    cout << "Witaj " << imie << "!" << endl;
    
    cout << "Ile masz lat? ";
    cin >> wiek;
    cout << "Urodziłeś się w " << rok-wiek << " roku" << endl; 
    
    if(wiek > wiek_cpp)
        cout << "Jestem młodszy o " << wiek-wiek_cpp << " lat" << endl;
    else if(wiek < wiek_cpp)
        cout << "Jestem starszy o " << wiek_cpp-wiek << " lat" << endl;
    else{
        cout << "Mamy tyle samo lat!" << endl;
    }
            
    cout << "Liczby parzyste (0-100): ";
    for(int i = 0; i < 101; i += 2){
		cout << i;
		if(i < 100) cout << ", ";
		else cout << "." << endl;
	}
    
   	return 0;
}
